README.md

```diff
@@ -43,3 +43,35 @@ let expected = vec![
 
 assert_eq!(resampled, expected);
 ```
+
+
+## Usage in Python
+
+To resample a stream of samples to a given interval, you can use the `Resampler`
+class.
+The construction of a resampler expects an interval (`datetime.timedelta`),
+a `ResamplingFunction`, a `max_age_in_intervals` parameter to control the
+maximum age of a sample (skips all samples if set to `0`), and a `start` parameter to set the start time of the
+first resampled sample.
+
+```python
+import datetime as dt
+from frequenz_resampling import Resampler, ResamplingFunction
+
+
+start = dt.datetime(1970, 1, 1, tzinfo=dt.timezone.utc)
+step = dt.timedelta(seconds=1)
+resampler = Resampler(dt.timedelta(seconds=5), ResamplingFunction.Average, 1, start)
+
+for i in range(10):
+    resampler.push(start + i * step, i + 1)
```

Comment:
Not sure if I wouldn't force keyword arguments here, this looks pretty obscure to me at least. Also maybe name it `push_sample()`?
```suggestion
    resampler.push_sample(timestamp=start + i * step, value=i + 1)
```

Reply:
Marking unresolved because it wasn't changed. We don't need to resolve it in the PR, but before resolving it we should at least create an issue about it.

Reply:
oh forgot the README. Changed it.